workflow/artifacts/oss/oss.go

```diff
@@ -462,6 +462,21 @@ func ListOssDirectory(bucket *oss.Bucket, objectKey string) (files []string, err
 	return files, nil
 }
 
-func (g *ArtifactDriver) IsDirectory(artifact *wfv1.Artifact) (bool, error) {
-	return false, errors.New(errors.CodeNotImplemented, "IsDirectory currently unimplemented for OSS")
+// IsDirectory tests if the key is acting like a OSS directory
+func (ossDriver *ArtifactDriver) IsDirectory(artifact *wfv1.Artifact) (bool, error) {
+	osscli, err := ossDriver.newOSSClient()
+	if err != nil {
+		return false, err
+	}
+	bucketName := artifact.OSS.Bucket
```

Comment:
Not convinced we need `bucketName` and `objectName` as variables here as they're initialized as simple copies and used once.

Reply:
ok, although it used once. but i want to be consistent with  "ListObjects".

Reply:
Ok, sounds reasonable.